From b3c9f9e91378e1f09cc9cb867e24c13677571cf8 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Thu, 2 Apr 2015 12:38:02 +0200
Subject: [PATCH] No Nicer error msg if stdlib pkg with cgo flag is not 
 installed

---
 hack/lib/golang.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hack/lib/golang.sh b/hack/lib/golang.sh
index 111a6a0..d315634 100644
--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -323,7 +323,6 @@ kube::golang::build_binaries() {
         local -a nonstatics=()
         for binary in "${binaries[@]}"; do
           if kube::golang::is_statically_linked_library "${binary}"; then
-            kube::golang::exit_if_stdlib_not_installed;
             statics+=($binary)
           else
             nonstatics+=($binary)
@@ -345,7 +344,6 @@ kube::golang::build_binaries() {
             fi
 
             if kube::golang::is_statically_linked_library "${binary}"; then
-              kube::golang::exit_if_stdlib_not_installed;
               CGO_ENABLED=0 go build -installsuffix cgo -o "${output_path}/${bin}" \
                 "${goflags[@]:+${goflags[@]}}" \
                 -ldflags "${version_ldflags}" \
-- 
1.9.3

