From 9e1cf168d92f155e478cd56c3b78ac4dee155237 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Tue, 7 Apr 2015 16:06:19 +0200
Subject: [PATCH] No Nicer error msg if stdlib pkg with cgo flag is not
 installed

---
 hack/lib/golang.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hack/lib/golang.sh b/hack/lib/golang.sh
index 5dc923f..588992b 100644
--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -276,7 +276,6 @@ kube::golang::build_binaries_for_platform() {
   local -a nonstatics=()
   for binary in "${binaries[@]}"; do
     if kube::golang::is_statically_linked_library "${binary}"; then
-      kube::golang::exit_if_stdlib_not_installed;
       statics+=($binary)
     else
       nonstatics+=($binary)
@@ -298,7 +297,6 @@ kube::golang::build_binaries_for_platform() {
       fi
 
       if kube::golang::is_statically_linked_library "${binary}"; then
-        kube::golang::exit_if_stdlib_not_installed;
         CGO_ENABLED=0 go build -installsuffix cgo -o "${output_path}/${bin}" \
           "${goflags[@]:+${goflags[@]}}" \
           -ldflags "${version_ldflags}" \
-- 
1.9.3

