From 49d9b8b5dd57db18c946f41715cff368c54339a9 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Tue, 11 Apr 2017 13:17:32 +0200
Subject: [PATCH] build with debug info

---
 hack/lib/golang.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hack/lib/golang.sh b/hack/lib/golang.sh
index c1259a4..9bf8243 100755
--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -508,7 +508,7 @@ kube::golang::build_binaries_for_platform() {
       CGO_ENABLED=0 "${GO}" build -o "${outfile}" \
         "${goflags[@]:+${goflags[@]}}" \
         -gcflags "${gogcflags}" \
-        -ldflags "${goldflags}" \
+        -ldflags "-B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n') ${goldflags}" \
         "${binary}"
       kube::log::progress "*"
     done
@@ -517,7 +517,7 @@ kube::golang::build_binaries_for_platform() {
       "${GO}" build -o "${outfile}" \
         "${goflags[@]:+${goflags[@]}}" \
         -gcflags "${gogcflags}" \
-        -ldflags "${goldflags}" \
+        -ldflags "-B 0x$(head -c20 /dev/urandom|od -An -tx1|tr -d ' \n') ${goldflags}" \
         "${binary}"
       kube::log::progress "*"
     done
-- 
2.7.4

