From 8e2484a6565ba7237156539ff679909fc1c9c44b Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Tue, 8 Nov 2016 14:12:48 +0100
Subject: [PATCH] fix support for ppc64le

---
 Makefile         | 2 +-
 hack/lib/init.sh | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index ac10e76..93ca680 100644
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ MAKEFLAGS += --no-builtin-rules
 # Constants used throughout.
 .EXPORT_ALL_VARIABLES:
 OUT_DIR ?= _output
-BIN_DIR := $(OUT_DIR)/bin
+BIN_DIR := $(OUT_DIR)/local/go/bin
 PRJ_SRC_PATH := k8s.io/kubernetes
 GENERATED_FILE_PREFIX := zz_generated.
 
diff --git a/hack/lib/init.sh b/hack/lib/init.sh
index d2b50d4..dfb4727 100644
--- a/hack/lib/init.sh
+++ b/hack/lib/init.sh
@@ -23,7 +23,7 @@ KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../.." && pwd -P)"
 
 KUBE_OUTPUT_SUBPATH="${KUBE_OUTPUT_SUBPATH:-_output/local}"
 KUBE_OUTPUT="${KUBE_ROOT}/${KUBE_OUTPUT_SUBPATH}"
-KUBE_OUTPUT_BINPATH="${KUBE_OUTPUT}/bin"
+KUBE_OUTPUT_BINPATH="${KUBE_OUTPUT}/go/bin"
 
 # Set no_proxy for localhost if behind a proxy, otherwise, 
 # the connections to localhost in scripts will time out
-- 
1.9.3

