From 15878d8a347f0dc2fd53abb348a3e9c38a9c61af Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Tue, 9 Aug 2016 18:33:54 +0200
Subject: [PATCH] hack hack/test-cmd.sh

---
 hack/lib/init.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hack/lib/init.sh b/hack/lib/init.sh
index 3464a96..c6446ac 100644
--- a/hack/lib/init.sh
+++ b/hack/lib/init.sh
@@ -41,7 +41,7 @@ source "${KUBE_ROOT}/hack/lib/etcd.sh"
 
 KUBE_GIT_UPSTREAM="${KUBE_GIT_UPSTREAM:-upstream}"
 
-KUBE_OUTPUT_HOSTBIN="${KUBE_OUTPUT_BINPATH}/$(kube::util::host_platform)"
+KUBE_OUTPUT_HOSTBIN="/usr/bin"
 
 # This emulates "readlink -f" which is not available on MacOS X.
 # Test:
-- 
1.9.3

