From bf75021c69915aa136ab83672edef5561367e09c Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Tue, 1 Mar 2016 09:25:59 +0100
Subject: [PATCH] s.Flags() does not carry all flags of individual options

---
 cmd/hyperkube/hyperkube.go | 12 ------------
 1 file changed, 12 deletions(-)

diff --git a/cmd/hyperkube/hyperkube.go b/cmd/hyperkube/hyperkube.go
index 5272c5d..87d8f68 100644
--- a/cmd/hyperkube/hyperkube.go
+++ b/cmd/hyperkube/hyperkube.go
@@ -154,18 +154,6 @@ func (hk *HyperKube) Run(args []string) error {
 		return err
 	}
 
-	s.Flags().AddFlagSet(hk.Flags())
-	err = s.Flags().Parse(args)
-	if err != nil || hk.helpFlagVal {
-		if err != nil {
-			hk.Printf("Error: %v\n\n", err)
-		}
-		s.Usage()
-		return err
-	}
-
-	verflag.PrintAndExitIfRequested()
-
 	util.InitLogs()
 	defer util.FlushLogs()
 
-- 
1.9.3

