From 5e6b6dcc870a7af2ca846596c1edac8fd525ce35 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Fri, 26 Feb 2016 09:55:47 +0100
Subject: [PATCH] un hyperkube apiserver

---
 cmd/hyperkube/main.go | 1 -
 1 file changed, 1 deletion(-)

diff --git a/cmd/hyperkube/main.go b/cmd/hyperkube/main.go
index 4d9c499..8b060ba 100644
--- a/cmd/hyperkube/main.go
+++ b/cmd/hyperkube/main.go
@@ -32,7 +32,6 @@ func main() {
 	}
 
 	hk.AddServer(NewKubectlServer())
-	hk.AddServer(NewKubeAPIServer())
 	hk.AddServer(NewKubeControllerManager())
 	hk.AddServer(NewScheduler())
 	hk.AddServer(NewKubelet())
-- 
1.9.3

