From 6c25e7295e1c57588964003c6bbc46e44a5e5a96 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Tue, 11 Apr 2017 13:28:53 +0200
Subject: [PATCH] use_go_build is not fully propagated so make it fixed for
 everyone

---
 hack/lib/golang.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hack/lib/golang.sh b/hack/lib/golang.sh
index 9bf8243..2806cb7 100755
--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -501,7 +501,7 @@ kube::golang::build_binaries_for_platform() {
       kube::golang::fallback_if_stdlib_not_installable;
   fi
 
-  if [[ -n ${use_go_build:-} ]]; then
+  if [[ True ]]; then
     kube::log::progress "    "
     for binary in "${statics[@]:+${statics[@]}}"; do
       local outfile=$(kube::golang::output_filename_for_binary "${binary}" "${platform}")
-- 
2.7.4

