From 8ddfd50cbaa61ab04685c7caf4b02bdeeefb9852 Mon Sep 17 00:00:00 2001
From: Jan Chaloupka <jchaloup@redhat.com>
Date: Mon, 20 Feb 2017 10:19:18 +0100
Subject: [PATCH] use_go_build is not fully propagated so make it fixed for
 everyone

---
 hack/lib/golang.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hack/lib/golang.sh b/hack/lib/golang.sh
index fbbd5e9..daab249 100755
--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -509,7 +509,7 @@ kube::golang::build_binaries_for_platform() {
     gogcflags="${gogcflags} -largemodel"
   fi
 
-  if [[ -n ${use_go_build:-} ]]; then
+  if [[ True ]]; then
     kube::log::progress "    "
     for binary in "${statics[@]:+${statics[@]}}"; do
       local outfile=$(kube::golang::output_filename_for_binary "${binary}" "${platform}")
-- 
2.7.4

